go/offline/rpc_cache.go

```diff
@@ -80,7 +82,7 @@ func (c *RPCCache) get(mctx libkb.MetaContext, version Version, rpcName string,
 	c.Lock()
 	defer c.Unlock()
 
-	dbk, err := dbKey(rpcName, arg)
+	dbk, err := dbKey(rpcName, mctx.G().GetMyUID(), arg)
```

Comment:
What about the case of no logged in uid?  Also, this might not give the right answer if the user is in the process of getting logged in ---- which I guess should never happen offline...

Reply:
that should be a separate namespace entirely and work fine with an empty UID -- shouldn't it?

Reply:
I think so, just 2x-checking

Reply:
cool. Do you think this is good to merge then?